lib/file-contents-eval.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var path = require('path');
+
+var nodeEval = require('node-eval');
+var JSON5 = require('json5');
```

Comment:
Run in 100000 files

Format | Real time | User Time | Sys Time
------- | --------- | ---------- | ----------
CommonJS | 4.237s | 3.709s | 1.359s
JSON | 4.240s | 3.724s | 1.367s
JSON 5 | 4.444s | 3.912s | 1.401s